### ISSUE TITLE
Not Working in IE 11

### ISSUE THREAD

**Issue by madceb** [2sic/2sxc#1212]
**I'm submitting a ...**  
```
- [ x ] bug report => search github for a similar issue before submitting 
- [ ] feature request
- [ ] support request => do not submit here, see https://2sxc.org/en/blog/post/moving-discussions-from-codeplex-to-stack-overflow
- [ ] not sure
```

**...about**  
```
- [ x ] edit experience / UI
- [ ] admin experience UI
- [ ] Razor templating
- [ ] JS development
- [ ] Content Types or data management
- [ ] APIs like REST
- [ ] DNN parts
- [ ] other / unknown
```

**Current behavior**

IE 11 javascript error in file inpage.min.js line 2029  if (modules.find(function (m) { Message: Object doesn't support property or method 'find', this breaks all 2sxc and other javascript related functionalities.

**Expected behavior**

In Chrome no javascript error shown and all functionalities work as expected, 2sxc and other javascript related functionalities.

**Minimal reproduction of the problem with instructions**

Open page containing 2sxc Content element with editor permissions in IE 11

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **2sxc version(s):** 9.2.0
* **Browser:** IE 11
* **DNN:** 9.1.0
* **Language:** Englsh/Spanish at least

**Anything you would like to add**

**Comment by iJungleboy** [comment]
@madceb thx for this report - we'll look into it and fix in 2sxc 9.3

**Comment by iJungleboy** [comment]
seems to be a duplicate of #1211 - already fixed :) ready in the upcoming release.
